base/ltfssaxes.dtx

```diff
@@ -264,7 +264,6 @@
 \DeclareFontSeriesChangeRule {ebc}{x}{ebx}{}
 %    \end{macrocode}
 %
-%
 %    \begin{macrocode}
 \DeclareFontSeriesChangeRule {ec}{bx} {bx}  {b} 	%<-----
```

Comment:
There are a number of entries of the form `\DeclareFontSeriesChangeRule{...}{...}{bx}{b}`. I suspect that they exist because for the Computer Modern fonts 'bold' is `bx` and not `b`, so that you have a reasonable substitution when `bx` is applied to another font. Wouldn't it be better if this was handled in a general way, e.g., in `\merge@font@series@`, instead of handling this case (incompletely) via random `\DeclareFontSeriesChangeRule` entries?